shared/translations/languages/de.json

```
@@ -672,11 +670,17 @@
     "Sent (Pending confirmation)": "Gesendet (Bestätigung ausstehend)",
     "Broadcasting...": "Übertrage...",
     "OK": "OK",
-    "Insufficient UTXO DFI": "Nicht ausreichend UTXO-DFI",
-    "Not enough balance": "Guthaben zu niedrig",
+    "Insufficient UTXO DFI.": "Nicht ausreichend UTXO-DFI.",
+    "Insufficient balance. Top up to proceed.": "Unzureichendes Guthaben. Aufstocken, um fortzufahren.",
     "Swap price is higher than the range allowed by the slippage tolerance. Increase tolerance percentage to proceed.": "Tauschpreis ist höher als die durch die Slippage-Toleranz erlaubte Spanne. Erhöhe den Toleranzprozentsatz, um fortzufahren.",
     "Preparing broadcast": "Übertragung wird vorbereitet",
-    "Waiting for transaction": "Warten auf Transaktion"
+    "Waiting for transaction": "Warten auf Transaktion",
+    "Insufficient DFI collateral (≥50%)": "Unzureichende DFI-Sicherheiten. (≥50%)",
```

think we need a full stop here? same for the original message in `TransactionError` and other `.json`
```suggestion
    "Insufficient DFI collateral. (≥50%)": "Unzureichende DFI-Sicherheiten. (≥50%)",
```